packages/api/src/utility.ts

```diff
@@ -284,22 +284,38 @@ export function checkSettingRestrictions(
   currentVal: unknown,
   restriction: SettingRestriction | undefined
 ) {
-  if (!restriction) return
+  if (!restriction) {
```

Comment:
Just some refactoring that happens when reading through code. Boy scout rule

`if`s without brackets and new lines etc. are super hard to read